src/modules/UI/components/OptionSubtext/OptionSubtext.ui.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React, {Component} from 'react'
+import T from '../FormattedText/FormattedText.ui'
+import styles from './style.js'
+import s from '../../../../locales/strings.js'
+import {sprintf} from 'sprintf-js'
+
+type Props = {
+  confirmationText: string,
+  label: string
+}
+
+type State = {}
+
+export default class OptionSubtext extends Component<Props, State> {
+  render () {
+    const WALLET_CONFIRMATION_TEXT = sprintf(this.props.confirmationText)
```

Comment:
if there are no dynamically generated strings to inject into the text, then `sprintf` is unnecessary